tests/components/zwave_js/test_api.py

```diff
@@ -325,6 +325,227 @@ async def test_remove_node(
     assert msg["error"]["code"] == ERR_NOT_LOADED
 
 
+async def test_replace_failed_node(
+    hass,
+    integration,
+    client,
+    hass_ws_client,
+    nortek_thermostat,
+    nortek_thermostat_added_event,
+    nortek_thermostat_removed_event,
+):
+    """Test the replace_failed_node websocket command."""
+    entry = integration
+    ws_client = await hass_ws_client(hass)
+
+    # Add mock node to controller
+    client.driver.controller.nodes[67] = nortek_thermostat
```

Comment:
The node is already added in the nodes dict in the fixture:
https://github.com/home-assistant/core/blob/331cb3b74da1caa6d6d6a86b17b20b2c6a9fbeb7/tests/components/zwave_js/conftest.py#L515